pkg/ui/src/views/login/loginPage.tsx

```diff
@@ -52,31 +63,74 @@ class LoginPage extends React.Component<LoginPageProps & WithRouterProps, LoginP
         });
   }
 
+  renderError() {
+    const { error } = this.props.loginState;
+
+    if (!error) {
+      return null;
+    }
+
+    let message = "Invalid username or password.";
+    if (error.message !== "Unauthorized") {
+        message = error.message;
+    }
+    return (
+      <div className="login-page__error">Unable to log in: { message }</div>
+    );
+  }
+
   render() {
+    const inputClasses = classNames("input-text", {
+      "input-text--error": !!this.props.loginState.error,
+    });
+
     return (
-      <div>
+      <div className="login-page">
         <Helmet>
           <title>Login</title>
         </Helmet>
-        <section className="section">
-          <h1>Login</h1>
-          {this.props.loginState.error
-            ? <div className="login-page__error">Login error: {this.props.loginState.error}</div>
-            : null}
-          <form onSubmit={this.handleSubmit}>
-            <input
-              type="text"
-              onChange={(evt) => this.handleUpdateUsername(evt.target.value)}
-              value={this.state.username}
-            /><br />
-            <input
-              type="password"
-              onChange={(evt) => this.handleUpdatePassword(evt.target.value)}
-              value={this.state.password}
-            /><br />
-            <input type="submit" disabled={this.props.loginState.inProgress} />
-          </form>
-        </section>
+        <div className="content">
+          <section className="section login-page__info">
+            <p className="version">
+              Version: <span className="version-tag">{ version }</span>
+            </p>
+            <img className="logo" alt="CockroachDB" src={logo} />
+            <p className="aside">
+              Please contact your database administrator for
+              account access and password restoration.
+            </p>
+            <p className="aside">
+              <a href={docsURL("admin-ui-overview.html")} className="docs-link">
```

Comment:
maybe change to https://www.cockroachlabs.com/docs/dev/admin-ui-access-and-navigate.html#secure-the-admin-ui in a followup?

Reply:
@Amruta-Ranade will this link change?